frank-doc-doclet/src/test/java/org/frankframework/frankdoc/wrapper/FrankClassDocletTest.java

```diff
@@ -38,4 +42,31 @@ public void testMethodSequenceIsPreserved() {
 		String[] expectedMethodNames = new String[] {"setInherited", "setVarargMethod", "getMyInnerEnum", "myAnnotatedMethod", "methodWithoutAnnotations"};
 		assertArrayEquals(expectedMethodNames, actualMethodNames.toArray(new String[] {}));
 	}
+
+	@Test
+	public void testExtendsOrInherits() throws Exception {
+		FrankClass grandChild = repository.findClass(PACKAGE + "GrandChild");
+		FrankClass child = repository.findClass(PACKAGE + "Child");
+		FrankClass parent = repository.findClass(PACKAGE + "Parent");
+		FrankClass myInterface = repository.findClass(PACKAGE + "MyInterface");
+		assertTrue(grandChild.extendsOrImplements(parent));
+		assertTrue(grandChild.extendsOrImplements(myInterface));
+		assertFalse(myInterface.extendsOrImplements(child));
+	}
+
+	@Test
+	public void testAnnotationHavingClassArrayValue() throws Exception {
+		FrankClass clazz = repository.findClass(PACKAGE + "WithAnnotationHavingClassArray");
+		FrankAnnotation annotation = clazz.getAnnotation(ANNOTATION_WITH_CLASS_ARRAY);
+		String[] value = (String[]) annotation.getValue();
+		assertArrayEquals(new String[] {PACKAGE + "Child", PACKAGE + "GrandChild"}, value);
+	}
+
+	@Test
+	public void testAnnotationHavingClassArrayValueOneValue() throws Exception {
+		FrankClass clazz = repository.findClass(PACKAGE + "WithAnnotationHavingClassArrayOneValue");
+		FrankAnnotation annotation = clazz.getAnnotation(ANNOTATION_WITH_CLASS_ARRAY);
+		String[] value = (String[]) annotation.getValue();
+		assertArrayEquals(new String[] {PACKAGE + "Child"}, value);
+	}
```

Comment:
Ik denk dat het verstandiger is deze test bij de ExcludeFromTypeFeatureTest te zetten. Is er een reden dat je de feature hier test?

Reply:
In ExcludeFromTypeFeatureTest werd al getest dat je het feature zowel met een JavaDoc tag als met een Java annotatie kan gebruiken. De bestaande tests namen het geval dat er maar één class gerefereerd wordt als waarde nog niet mee. Die test heb ik nu toegevoegd, maar alleen met een Java annotatie.